src/panels/my/ha-panel-my.ts

```diff
@@ -432,6 +432,9 @@ class HaPanelMy extends LitElement {
             >`,
           });
           break;
+        case "url_error":
+          error = this.hass.localize("ui.panel.my.url_error");
+          break;
         default:
```

Comment:
If they'll come up with a new type of error we don't support, it will reach default.

Reply:
😕 "They" meaning users? The `_error` property is fixed by the class, and it would be cleaner here to just assign it the localized string rather than dealing with an intermediate key.

Reply:
Oh, you're right. I've removed the switch.

Reply:
Are we sure translations are loaded when we assign the error now? As before it would be updated in render.

Reply:
Actually, I don't think we can assume that, however, I didn't see translations being broken during testing. 
I guess I will have to add it back?